tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "./tsconfig.base.json",
   "files": [],
+  "compilerOptions": {
+    "lib": ["esnext"],
+  },
```

Comment:
Please revert this change: https://nx.dev/docs/concepts/typescript-project-linking#set-up-typescript-project-references